docs/submit/install-ospool-ap.md

```diff
@@ -177,20 +195,20 @@ the [Organizing and Submitting HTC Workloads Tutorial](https://portal.osg-htc.or
 are particularly relevant.
 
 ### Specifying a project
-OSG will only run jobs that have a registered *project* associated with them.
+OSPool resources will only run jobs that have a registered *project* associated with them.
```

Comment:
Capacity?

Reply:
"Capacity" is not a thing that can run jobs -- capacity is more like a slice of time of some resources. I think EP would be the best term here.